### PR TITLE
Add remove_circle to default sprite

### LINKED ISSUE #4445: Add `Remove Circle` icon to sprite
ZenDesk user asked us to consider adding the "Remove Circle" icon to our library: https://fonts.google.com/icons?selected=Material+Icons:remove_circle:

### PR DESCRIPTION
**Pull request by thisisdano**
This PR adds the `remove_circle` icon to the default sprite with the meta tags `delete` and `minus`.

Fixes #4445 

Screen Shot 2022-06-08 at 4 39 29 PM

[Preview TK]() →

### CODE REVIEW

**Comment by thisisdano** [comment]
Hold this one until we merge https://github.com/uswds/uswds/pull/4794 — the process is changing for how we add new icons to the sprite

**Review by mejiaj — APPROVED** [review]

**Pull request merged**
